Models/fetchTopics.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+function fetchTopics(request, response, next) {
```

Comment:
Next isn't accessible in the model - only the controller. As it's the last parameter in the function and we aren't passing anything in or trying to access it - it won't do any harm, but nor will it achieve anything. Please remove since we shouldn't have unused code pushed to github